src/app/containers/CpsAssetPageMain/__snapshots__/index.test.jsx.snap

```diff
@@ -1,94 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`CpsAssetPageMain should match snapshot 1`] = `
```

Comment:
TODO: Check why a lot of CSS classes went missing? Is this expected?

Reply:
I think they have been replaced by the snapshots in https://github.com/bbc/simorgh/blob/latest/src/app/lib/styledGrid/__snapshots__/index.test.jsx.snap when you updated the snapshots. `GridItemConstrainedMedium` is being used in CpsAssetPageMain. https://github.com/bbc/simorgh/blob/latest/src/app/lib/styledGrid/index.jsx#L45

Reply:
Yeah it was because I made a temporary function so the it block could be async. Now that I've added that functionality to psammead-test-helpers@3.1.1 this is working as expected.